Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/hibernate/entity/User.java

```diff
@@ -0,0 +1,115 @@
+package hibernate.entity;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+public class User implements Comparable<User>{
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private int id;
+
+    @Column(name="login")
+    private String login;
+
+    @Column(name="password")
+    private String password;
+
+    @Temporal(TemporalType.DATE)
+    @Column(name = "regdate")
+    java.util.Date dateOfRegistration;
```

Comment:
You use another date class here?

Reply:
Yes, it's convenient.
Because @Temporal(TemporalType.DATE) drops the time value and only preserves the date.
The time data can have the accuracy of DATE, TIME or TIMESTAMP (ie Actual date, time only, or both). For fine tuning, use the annotation.

Reply:
Question is not about Temporal. Question was about java.util.Date, why not just Date?

Reply:
Got it. Yes, it is now unnecessary. The Date class has many variations in different libraries. In order not to accidentally import another, I explicitly indicated it. Deleted on local computer